tests/deepsparse/pipelines/test_clip.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from deepsparse import BasePipeline, Pipeline
+from deepsparse.clip import (
+    CLIPTextInput,
+    CLIPTextOutput,
+    CLIPTextPipeline,
+    CLIPVisualInput,
+    CLIPVisualOutput,
+    CLIPVisualPipeline,
+    CLIPZeroShotInput,
+    CLIPZeroShotOutput,
+    CLIPZeroShotPipeline,
+)
+from tests.deepsparse.pipelines.data_helpers import computer_vision
+from tests.utils import mock_engine
+
+
+@pytest.fixture
+def visual_input():
+    images = computer_vision(batch_size=2)
+    model_path = "clip_onnx/clip_visual.onnx"
```

Comment:
This is meant to be replaced with sparsezoo stub?

Reply:
Yes, the tests are all skipped right now as they aren't yet in sparsezoo. MLR is working on that now so will update paths in a follow-up PR once they've added them